rhel/SPECS/dbaas-tools.spec

```diff
@@ -3,26 +3,31 @@
 %global commit_aws          d7c0b2e9131faabb2b09dd804a35ee03822f8447
 %global shortcommit_aws     %(c=%{commit_aws}; echo ${c:0:7})
 
-%global commit_k8s          ec6eb119b81be488b030e849b9e64fda4caaf33c
-%global shortcommit_k8s     %(c=%{commit_k8s}; echo ${c:0:7})
-%global version_k8s         v1.16.8
+%global commit_k8s_116          ec6eb119b81be488b030e849b9e64fda4caaf33c
+%global shortcommit_k8s_116     %(c=%{commit_k8s_116}; echo ${c:0:7})
+%global version_k8s_116         v1.16.8
+
+%global commit_k8s_121          5e58841cce77d4bc13713ad2b91fa0d961e69192
+%global shortcommit_k8s_121     %(c=%{commit_k8s_121}; echo ${c:0:7})
+%global version_k8s_121         v1.21.1
 
 %global install_golang 1
 
 %define build_timestamp %(date -u +"%y%m%d%H%M")
-%define release         4
+%define release         1
 %define rpm_release     %{release}.%{build_timestamp}%{?dist}
 
 Name:           dbaas-tools
-Version:        0.5.1
+Version:        0.6.0
 Release:        %{rpm_release}
 Summary:        A set of tools for Percona DBaaS
 License:        ASL 2.0
 URL:            https://github.com/kubernetes-sigs/aws-iam-authenticator
 # Git tag can be moved and pointed to different commit hash which may brake reproducibility of the build
 # As by using an exact commit hash, we can ensure that each time source will be identical
 Source0:        https://github.com/kubernetes-sigs/aws-iam-authenticator/archive/%{commit_aws}/aws-iam-authenticator-%{shortcommit_aws}.tar.gz
-Source1:        https://github.com/kubernetes/kubernetes/archive/%{commit_k8s}/kubernetes-%{shortcommit_k8s}.tar.gz
+Source1:        https://github.com/kubernetes/kubernetes/archive/%{commit_k8s_116}/kubernetes-%{shortcommit_k8s_116}.tar.gz
+Source2:        https://github.com/kubernetes/kubernetes/archive/%{commit_k8s_121}/kubernetes-%{shortcommit_k8s_121}.tar.gz
 
 %if %{install_golang}
 BuildRequires:   golang >= 1.13.0
```

Comment:
Should we bump up the version of go as well?

Reply:
No, it's not necessary